src/main/kotlin/week1/calculator/Calculator.kt

```diff
@@ -0,0 +1,33 @@
+package week1.calculator
+
+object Calculator {
+
+    private const val DELIMITER = ' '
+
+    fun process(input: String?): Int {
+        require(!input.isNullOrBlank()) {
+            if (input == null) "식이 null입니다." else "비어 있는 식입니다."
+        }
```

Comment:
예외 메시지를 분기처리하여 출력하는 것보다는 아래와 같은 예외메시지로 변경하여 input으로 어떤 값이 들어오는지를 예외메시지에 포함시키면 왜 예외가 발생했는지 원인 파악이 쉬울 것 같아요. 😃
```suggestion
        require(!input.isNullOrBlank()) {  "입력 값은 null 또는 빈 문자열을 허용하지 않습니다. input: $input" }
```